traffic_router/core/src/main/java/org/apache/traffic_control/traffic_router/core/config/ConfigHandler.java

```diff
@@ -15,65 +15,41 @@
-
-import org.apache.traffic_control.traffic_router.core.secure.CertificatesPoller;
-import org.apache.traffic_control.traffic_router.core.secure.CertificatesPublisher;
-import org.apache.traffic_control.traffic_router.core.util.JsonUtils;
-import org.apache.traffic_control.traffic_router.core.util.JsonUtilsException;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-
-import org.apache.traffic_control.traffic_router.core.edge.Cache.DeliveryServiceReference;
-import org.apache.traffic_control.traffic_router.core.edge.Cache;
-import org.apache.traffic_control.traffic_router.core.edge.CacheLocation;
-import org.apache.traffic_control.traffic_router.core.edge.Location;
-import org.apache.traffic_control.traffic_router.core.edge.CacheRegister;
-import org.apache.traffic_control.traffic_router.core.edge.Node;
-import org.apache.traffic_control.traffic_router.core.edge.TrafficRouterLocation;
 import org.apache.traffic_control.traffic_router.core.ds.DeliveryService;
 import org.apache.traffic_control.traffic_router.core.ds.DeliveryServiceMatcher;
 import org.apache.traffic_control.traffic_router.core.ds.DeliveryServiceMatcher.Type;
+import org.apache.traffic_control.traffic_router.core.ds.LetsEncryptDnsChallengeWatcher;
+import org.apache.traffic_control.traffic_router.core.ds.SteeringWatcher;
+import org.apache.traffic_control.traffic_router.core.edge.*;
+import org.apache.traffic_control.traffic_router.core.edge.Cache.DeliveryServiceReference;
+import org.apache.traffic_control.traffic_router.core.loc.*;
 import org.apache.traffic_control.traffic_router.core.monitor.TrafficMonitorWatcher;
+import org.apache.traffic_control.traffic_router.core.request.HTTPRequest;
+import org.apache.traffic_control.traffic_router.core.router.StatTracker;
 import org.apache.traffic_control.traffic_router.core.router.TrafficRouterManager;
+import org.apache.traffic_control.traffic_router.core.secure.CertificatesPoller;
+import org.apache.traffic_control.traffic_router.core.secure.CertificatesPublisher;
+import org.apache.traffic_control.traffic_router.core.util.JsonUtils;
+import org.apache.traffic_control.traffic_router.core.util.JsonUtilsException;
 import org.apache.traffic_control.traffic_router.core.util.TrafficOpsUtils;
-import org.apache.traffic_control.traffic_router.core.router.StatTracker;
 import org.apache.traffic_control.traffic_router.geolocation.Geolocation;
-import org.apache.traffic_control.traffic_router.core.request.HTTPRequest;
-import org.apache.traffic_control.traffic_router.core.loc.AnonymousIp;
-import org.apache.traffic_control.traffic_router.core.loc.AnonymousIpConfigUpdater;
-import org.apache.traffic_control.traffic_router.core.loc.AnonymousIpDatabaseUpdater;
+
+import java.io.IOException;
+import java.net.URL;
+import java.net.UnknownHostException;
+import java.util.*;
```

Comment:
The TR code conventions include expanded imports, no wildcard imports

Reply:
Done